util/util.go

```diff
@@ -0,0 +1,5 @@
+package util
+
+func StringToPointer(str string) *string {
```

Comment:
Can you remove this function and use [this one](https://github.com/asticode/go-astikit/blob/master/ptr.go#L41) instead (`astikit` is already used in the lib)

Reply:
done